io.openems.edge.greenconsumptionadvisor.api/src/io/openems/edge/greenconsumptionadvisor/api/GreenConsumptionAdvisorImpl.java

```diff
@@ -0,0 +1,193 @@
+package io.openems.edge.greenconsumptionadvisor.api;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
+import java.time.temporal.ChronoUnit;
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
```

Comment:
We have a "HttpBridge" for this i guess

Reply:
Hi Sn0wy, thanks for your review and your comments!
As I understood it, the HttpBridge is executed every cycle. Since the data is only updated every hour, the https request is currently only executed every hour. Did I get it wrong?

Reply:
True, you still could implement a Counter for the requests :)

Reply:
Sorry I guess i can't follow. Could you please explain it a bit more? So you mean using the httpBridge.subscribeJsonEveryCycle(); function and adding a counter its passed BiConsumer?